### PR TITLE
Clean up media elements to avoid leak

### PR DESCRIPTION
**Pull request by arunpbk-stayqrious**
Chrome 92 limits media elements created in a frame.

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
Source: https://twitter.com/quicksave2k/status/1382247766391394304

**Comment by gkatsev** [comment]
Yeah, noticed this in our playback engine tests for Video.js (VHS).
It makes sense to clean up after ourselves anyway.

**Review by gkatsev — APPROVED** [review]

File: lib/index.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ function startPlayback ({muted, timeout, inline}, elementCallback) {
     } else {
       sendOutput(true)
     }
+    
+    // Clean up to avoid MediaElementLeak
+    element.remove();
+    element.srcObject = null;
```
Comment (gkatsev) [inline lib/index.js:50]: nitpick: looks like this project is currently set up with `standard`, so, trailing semicolons aren't allowed.
```suggestion
    element.remove()
    element.srcObject = null
```

**Review by gkatsev — COMMENTED** [review]
Tests pass locally, so, looks good to me.

**Comment by gkatsev** [comment]
@caiogondim @NicolasSiver if one of you can add me on npm, I'd be happy to take care of publishing this once it's merged.

File: lib/index.js [hunk 2]
```diff
@@ -44,6 +44,10 @@ function startPlayback ({muted, timeout, inline}, elementCallback) {
     } else {
       sendOutput(true)
     }
+
```
Comment (gkatsev) [inline lib/index.js:47]:
```suggestion

```

**Comment by gkatsev** [comment]
Published as 3.0.1!

**Pull request merged**